taskcluster/rb_taskgraph/build.py

```diff
@@ -42,3 +44,12 @@ def add_variant_config(config, tasks):
                 apks[apk["abi"]] = apk_name
 
         yield task
+
+
+@transforms.add
+def populate_version_name(config, tasks):
+    for task in tasks:
+        if task.getdefault("attributes", {})["nightly"]:
+            formatted_date = datetime.datetime.now().strftime('%y%V')
```

Comment:
What's `%V` ? I got `1934` for this, where 19 is the last 2 digits of 2019. Not sure where the `34` came from. Is that what you intended?

Reply:
`%V` is the current week of the year.
I just want to maintain backwards compatibility with [the version string that was generated before](https://github.com/mozilla-mobile/reference-browser/blob/master/automation/taskcluster/decision_task.py#L124).